docs/contributing-via-pr.md

```diff
@@ -36,4 +36,5 @@ You can post the following comment/command:
 Command | Functionality |
 --- | --- |
 `/pipeline restart community-hosted-pipeline` | The hosted pipeline will be re-triggered and PR will be merged if possible. |
-`/pipeline restart community-release-pipeline` | The release pipeline will be re-triggered.
\ No newline at end of file
+`/pipeline restart community-release-pipeline` | The release pipeline will be re-triggered.
+`/test skip {test_case_name}` | *test_case_name* test will be skipped.
```

Comment:
Can you please mention here that only a subset of tests (currently only pruned graph test) can be skipped? Other tests are still required.

Reply:
Sure. PR updated.